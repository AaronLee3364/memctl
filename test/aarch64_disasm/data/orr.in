fffffff0070c0000 2a140109 aa0ac108 3219006b
