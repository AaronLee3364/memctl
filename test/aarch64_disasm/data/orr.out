fffffff0070c0000  ORR     W9, W8, W20
fffffff0070c0004  ORR     X8, X8, X10, LSL #48
fffffff0070c0008  ORR     W11, W3, #0x80
